algoliasearch-common/src/main/java/com/algolia/search/responses/SearchQueryRuleResult.java

```diff
@@ -0,0 +1,49 @@
+package com.algolia.search.responses;
+
+import com.algolia.search.inputs.query_rules.QueryRule;
+
+import java.util.List;
+
+public class SearchQueryRuleResult {
```

Comment:
I know we use the same pattern for the regular search, but I am concerned about forward-compatibility here. Let's say we add another field in the response that is not supported by this class: what will happen? Parse error? (For example, we could add debug fields if certain options are activated.)